### PR TITLE
Completed javascript-methods exercise - Justin Song

### PR DESCRIPTION
**Pull request by justinsong1608**
Screenshot 2023-02-01 at 9 38 05 AM

### CODE REVIEW

File: javascript-methods/main.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  author: 'Dr. Axel Rauschmayer'
+};
+var css = {
+  title: 'CSS Secrets',
+  author: 'Lea Verou'
+};
+
+library.push(js);
+library.unshift(css);
+library.splice(1);
```
Comment (shawnkost) [inline javascript-methods/main.js]: You're missing a 2nd argument here. Right now you're deleting everything from the array after the 1st index. Check your console log and compare it to the example output

**Review by shawnkost — CHANGES_REQUESTED** [review]
One small change requested here Justin

File: javascript-methods/main.js (hunk 1 above)
Comment (justinsong1608) [reply javascript-methods/main.js]: Screenshot 2023-02-01 at 10 05 49 AM

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**